tests/suites/test_suite_gcm.function

```diff
@@ -8,6 +8,49 @@
  */
 
 /* BEGIN_CASE */
+void gcm_bad_parameters(  int cipher_id, int direction,
+                          char *hex_key_string, char *hex_src_string,
+                          char *hex_iv_string, char *hex_add_string,
+                          int tag_len_bits, int  gcm_result)
+{
+    unsigned char key_str[128];
+    unsigned char src_str[128];
+    unsigned char dst_str[257];
+    unsigned char iv_str[128];
+    unsigned char add_str[128];
+    unsigned char tag_str[128];
+    unsigned char output[128];
+    unsigned char tag_output[16];
+    mbedtls_gcm_context ctx;
+    unsigned int key_len;
+    size_t pt_len, iv_len, add_len, tag_len = tag_len_bits / 8;
+
+    mbedtls_gcm_init( &ctx );
+
+    memset(key_str, 0x00, 128);
+    memset(src_str, 0x00, 128);
+    memset(dst_str, 0x00, 257);
+    memset(iv_str, 0x00, 128);
+    memset(add_str, 0x00, 128);
+    memset(tag_str, 0x00, 128);
+    memset(output, 0x00, 128);
+    memset(tag_output, 0x00, 16);
```

Comment:
Instead of hardcoding the last input parameter for `memset()` perhaps it would be good to use `sizeof()`?